### ISSUE TITLE
Dark Mode feature

### ISSUE THREAD

**Issue by aditya-8-88** [dguo/make-a-readme#89]
I think it should also have a dark mode feature.

**Comment by aditya-8-88** [comment]
@dguo are you there

**Comment by dguo** [comment]
Yep! I agree it'd be nice to have a dark mode. I probably won't get around to implementing it anytime soon, but if someone else would like to submit a PR for it before I work on it, that'd be great!

**Comment by aditya-8-88** [comment]
I have got some time.
